ensemblrest/ensemblrest.py

```diff
@@ -408,9 +411,13 @@ def __retry_request(self):
             message = ensembl_http_status_codes[self.last_response.status_code][1]
             
             # parse error if possible
-            json_message = json.loads(self.last_response.text)
-            if "error" in json_message:
-                message = json_message["error"]
+            try:
+                json_message = json.loads(self.last_response.text)
+                if "error" in json_message:
+                    message = json_message["error"]
+            except ValueError:
+                # In this case we didn't even get a JSON back.
+                message = "Server returned invalid JSON."
```

Comment:
Seems reasonable. However, is this called? could you check this?

Reply:
Will try to cover this in the next version of testing.